packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -46,6 +46,7 @@ import { createGetObjectsOfType, createSelector, isAdmin } from 'selectors'
 // Button's height = react-select's height(36 px) + react-select's border-width(1 px) * 2
 // https://github.com/JedWatson/react-select/blob/916ab0e62fc7394be8e24f22251c399a68de8b1c/less/select.less#L21, L22
 const SHARE_BUTTON_STYLE = { height: '38px' }
+const HA_VALUES = ['Restart', 'Restart if possible', 'Disable']
```

Comment:
We should probably expose the values in `xo-server` kind of like we did here:
https://github.com/vatesfr/xen-orchestra/blob/fdde916388665b9af72688196b8e6bd7079fe08f/packages/xo-server/src/api/network.js#L3-L5
and then handle the `''` case with an intl message.